fetchy/fetchyboi.ts

```diff
@@ -65,4 +82,12 @@ const main = async () => {
   }
 };
 
+const main = async () => {
+  if (process.env.REGEN_DATA === "true") {
```

Comment:
Can we possibly be really explicit with this variable?

```suggestion
  if (process.env.REGENERATE_DATA === "true") {
```

Reply:
sweet will do agreed.